types/emartech__cls-adapter/emartech__cls-adapter-tests.ts

```diff
@@ -0,0 +1,21 @@
+import ClsAdapter from '@emartech/cls-adapter';
```

Comment:
```suggestion
import ClsAdapter = require('@emartech/cls-adapter');
```
(when you appy this removal of `esModuleInterop`)